components/formats-bsd/src/loci/formats/in/MicromanagerReader.java

```diff
@@ -718,6 +718,13 @@ else if (key.equals("Frames")) {
         else if (key.equals("Slices")) {
           ms.sizeZ = Integer.parseInt(value);
         }
+        else if (key.equals("SlicesFirst")) {
+          if (value.equals("false")) {
+            ms.dimensionOrder = "XYCZT";
+          } else {
+            ms.dimensionOrder = "XYZCT";
+          }
```

Comment:
This conditional statement seems to be setting the same dimension order either way